tests/common/dualtor/dual_tor_utils.py

```diff
@@ -185,19 +185,31 @@ def update_mux_configs_and_config_reload(dut, state):
     dut.file(path=TMP_FILE, state='absent')
 
 
-def force_active_tor(dut, intf):
+@pytest.fixture
+def force_active_tor():
     """
     @summary: Manually set dut host to the active tor for intf
     @param dut: The duthost for which to toggle mux
     @param intf: One or a list of names of interface or 'all' for all interfaces
     """
-    if type(intf) == str:
-        cmds = ["config muxcable mode active {}".format(intf)]
-    else:
-        cmds = []
-        for i in intf:
-            cmds.append("config muxcable mode active {}".format(i))
-    dut.shell_cmds(cmds=cmds)
+    forced_intfs = []
+    def _force_active_tor(dut, intf):
```

Comment:
Minor - to me it seems counterintuitive to define an internal function (leading underscore), and then yield it too for external use :)

Reply:
True, though I think we already do it elsewhere so I just followed that convention. I'll change it